### DISCUSSION TITLE
Show tooltip with speaker name over icon

### DISCUSSION THREAD

**Discussion by fliespl** [music-assistant/support#1286] [Feature requests and ideas]
https://ss.codeone.pl/ss-2023-06-04-23-12-24-1685913144-P9rkBQlo.png

Allows to check if you are actually using wanted speaker.

**Comment by OzGav** [comment]
There is a major frontend redesign underway. Try and remember to bring this up again if it is still a problem when this is released in a couple of months.

**Reply by OzGav** [reply]
Just to round this out. The player name is shown in the control bar not as a tooltip 
![image](https://github.com/user-attachments/assets/c3df04d5-5845-467e-ac63-95a2cc9a8ea9)
